Plugin/Gateway.php

```diff
@@ -57,10 +57,10 @@ public function createPayment($data) {
 
 		try {
 			$response = $this->client->execute();
-
-			return $response['return'];
+			if ($response['return'] === 'API key not valid') $response['errors'] = 'true';
```

Comment:
I would prefer to put these errors message inside a `const`. Also, can't you just use `true` instead of `'true'`?

Reply:
the api also return `errors` as string

Reply:
will put them as a const in the class

Reply:
That should not be the case. We don't return the key `errors` in the API. This is the response with an invalid API key:

```
{
    "return": "API key not valid"
}
```

Therefore I think you can safely set the value to `true` as a `boolean` and use that.


The error messages should be constants inside the class.

Reply:
yeah but other responses do return it as a string

Reply:
I see we do return `error` when the error message is `Not subscribed to money transfer service`. You can ignore that comment and keep it as string for now.

Reply:
added them as a const in the class